pkg/netceptor/conn.go

```diff
@@ -45,12 +45,12 @@ type AcceptResult struct {
 
 // Listener implements the net.Listener interface via the Receptor network.
 type Listener struct {
-	s          *Netceptor
-	pc         PacketConner
-	ql         *quic.Listener
+	S          *Netceptor
```

Comment:
I think I would prefer creating some `Getters()` and `Setters()` rather than exposing the private data fields, 

I can see you have also created a new constructor, so its just continuing the same idea